viam_cartographer.go

```diff
@@ -123,23 +123,30 @@ func initSensorProcesses(cancelCtx context.Context, cartoSvc *CartographerServic
 		InternalTimeout:          cartoSvc.cartoFacadeInternalTimeout,
 		Logger:                   cartoSvc.logger,
 		RunFinalOptimizationFunc: cartoSvc.cartofacade.RunFinalOptimization,
-		Mutex:                    &sync.Mutex{},
 	}
 
-	cartoSvc.sensorProcessWorkers.Add(1)
-	go func() {
-		defer cartoSvc.sensorProcessWorkers.Done()
-		if jobDone := spConfig.StartLidar(cancelCtx); jobDone {
-			cartoSvc.jobDone.Store(true)
-			cartoSvc.cancelSensorProcessFunc()
-		}
-	}()
+	if spConfig.IsOnline {
```

Comment:
Can we add a comment here to state that online mode is parallelized and offline is sequential?